### ISSUE TITLE
i18n-related QA

### ISSUE THREAD

**Issue by k88hudson** [mozilla/activity-stream#2027]
Split from #1981. Review UI for: 
- quality of template strings
- languages with longer strings  (e.g. German)
- languages with shorter strings (e.g. Mandarin)
- RTL languages

**Comment by flodolo** [comment]
A few montes on the strings landed in #2110 

```
"bookmark": "Bookmark",
```
This is a tricky one: is it a verb (e.g. a button label, "add to bookmark") or a noun ("a bookmark")? We'll need comments for cases like this, and it looks like this format doesn't provide them.

```
"email_link": "Email Link...",
"loading": "Loading...",
```
You should use the unicode … (that's the character used in Firefox), unless there are technical limitations.

```
"search_for": "Search for {search_term} with",
```
This assumes that all languages can use this same grammatical structure as English. The name of the search provider should be a placeholder.

For reference, this is the string in Firefox
https://hg.mozilla.org/releases/comm-aurora/file/default/suite/locales/en-US/chrome/browser/navigator.properties#l45

If you’re doing something fancy, like using an image for the search engine, there should be a clear explanation in a comment.

**Comment by k88hudson** [comment]
Thanks for the feedback @flodolo!

**Comment by k88hudson** [comment]
Feedback addressed in  f1e82ec3cebb811d2551e067ecf59dafd5d134b8 @sarracini sarracini committed 5 days ago

**Comment by Mardak** [comment]
No formal review step needed, so we'll untrack this issue.
